pkg/tests/helpers_test.py

```diff
@@ -27,20 +28,20 @@ def testStripped(self):
     self.assertEqual(helpers.GetFlagValue('value ', strip=True), 'value')
 
   def testNonStripped_fromFile(self):
-    with tempfile.NamedTemporaryFile() as f:
-      f.write(b'value ')
-      f.flush()
-      self.assertEqual(helpers.GetFlagValue(
-          '@{}'.format(f.name),
-          strip=False), 'value ')
+    with tempfile.TemporaryDirectory() as temp_d:
```

Comment:
@aiuto I don't know if tests are supposed to be 2-3 cross-compatible, but FYI `TemporaryDirectory` is only present since 3.2, see [ref](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory)